lms/static/sass/lms-course.scss

```diff
@@ -24,3 +24,11 @@
         }
     }
 }
+
+.MathJax>svg {
```

Comment:
I recommend using HTML CSS instead of SVG whenever possible. SVG is challenging for [MathJax plugin makers](https://github.com/OmarIthawi/arabic-mathjax) like myself when it comes to Arabic and RTL support.

Additionally, Open edX needs to support mobile and to stay accessible which I think MathJax SVG isn't very well suited for:

 - https://docs.mathjax.org/en/latest/output/html.html